### ISSUE TITLE
Lyra without android support 

### ISSUE THREAD

**Issue by shoham5** [google/lyra#68]
Hi,
 
could I use Lyra on Linux-base-target without android support? 

I have seen in [WORKSPACE](https://github.com/google/lyra/blob/main/WORKSPACE) android_sdk_repository and android_ndk_repository dependencies. also remember that I've got an error when using other version on Linux.    

Thanks,

**Comment by aluebs** [comment]
Absolutely! If you build Lyra on Linux (and don't add the `--config=android_arm64` flag), it will build a Linux target.
If yo are still experiencing errors, please share the error message, so we can assist you better.

**Comment by aluebs** [comment]
This bug should be fixed in the new [Lyra 1.2.0](https://github.com/google/lyra/releases/tag/v1.2.0). Please let us know if you still encounter these issues.
